src/pages/Register/index.jsx

```diff
@@ -6,6 +6,7 @@ import Authenticate from '../../components/Authenticate'
 import { disableButton } from '../../store/reducers/buttonState'
 import { Form, FormCheckBox, FormField, SubmitButton } from '../../components/Form'
 import { registerUser } from '../../store/reducers/user'
+import { useQuery } from '../../utils/query';
```

Comment:
this is not a blocker though, i can approve this... But check this out for knowledge base.

You should be able to merge.

Reply:
Ooh, so what I did is to pass the affiliate code as query params instead of path parameter. Though there is a library that does this I feel having to install a package that achieves this hook is not necessary.
Thanks for the feedback btw.